src/frontend/src/expr/mod.rs

```diff
@@ -240,7 +241,9 @@ impl ExprImpl {
     /// Evaluate a constant expression.
     pub fn eval_row_const(&self) -> RwResult<Datum> {
         assert!(self.is_const());
-        tokio::runtime::Handle::current().block_on(self.eval_row(&OwnedRow::empty()))
+        self.eval_row(&OwnedRow::empty())
+            .now_or_never()
+            .expect("constant expression should not be async")
```

Comment:
If we need to do this frequently in the future, how about making it an extension to the `Expo` trait?

Reply:
It's okay. But I don't think this should be encouraged. It should only be used in constant evaluation. 🤔